docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -495,7 +495,7 @@ simulants are moving at the expected rates.
 2.3.5 Component 9: Address
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Each household id should be associated with a postal address, and when
+Each household id should be associated with a residential address, and when
```

Comment:
In our first pass, people will always move into brand-new addresses created by `faker` / no one will ever move into a previously vacated address. (Unless you'd like to change this)

Reply:
not necess yet

Reply:
Got it. would you mind updating this doc to reflect that so it matches the current implementation?

Reply:
done